kitops/cli/kit.py

```diff
@@ -1,10 +1,57 @@
+'''
+Copyright 2024 The KitOps Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+'''
+
 import json
+import os
 import subprocess
 import yaml
 from typing import Any, Dict, List, Optional
+from ..modelkit.kitfile import Kitfile
 from ..modelkit.utils import Color, IS_A_TTY
 from .utils import _process_command_flags
 
+def import_from_hf(repo_path_without_tag: str, **kwargs) -> None:
+    """
+    Import a model from HuggingFace. Download a repository from HuggingFace and 
+    package it as a ModelKit.
+    
+    The repository can be specified either via a repository (e.g. myorg/myrepo) or
+    with a full URL (https://huggingface.co/myorg/myrepo). The repository will be
+    downloaded to a temporary directory and be packaged using a generated Kitfile.
+    
+    Note: importing repositories requires 'git' and 'git-lfs' to be installed.
+
+    Args:
+        repo_path_with_tag (str): The repository path along with the tag to be used for the package.
+        **kwargs: Additional arguments to pass to the command.
+
+    Returns:
+        None
+        
+    Raises:
+        subprocess.CalledProcessError: If the command returns a non-zero exit status,
+        The exception contains the return code and the standard error output.
+    """
+    command = ["kit", "import", repo_path_without_tag]
+
+    command.extend(_process_command_flags(kit_cmd_name="import", **kwargs))
+    result = _run(command=command, input="n\n")
```

Comment:
Alternatively, if you're able to close stdin for the command you can avoid being prompted. E.g. in terminal
```bash
<&- kit import <repo>
```
will disable prompts